libs/designsystem/src/lib/components/action-sheet/action-sheet.component.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../scss/utils';
+
+:host {
+  position: relative;
+  display: inline-block;
+  outline: none;
+}
+
+:host(.is-open) {
+  kirby-action-sheet-popout {
+    display: block;
+  }
+  button {
+    box-shadow: get-elevation(8);
```

Comment:
@bommariusser Where can I find the action sheet button desktop example in Zeplin? Is this the correct elevation?